pkg/apis/machine/v1beta1/condition_consts.go

```diff
@@ -27,3 +27,18 @@ const (
 	// from making any further remediations.
 	TooManyUnhealthyReason = "TooManyUnhealthy"
 )
+
+const (
+	// MachineExistsCondition is set on the Machine to show whether a virtual mahcine has been created by the cloud provider.
+	MachineExistsCondition ConditionType = "MachineExists"
```

Comment:
I'm bit concerned about mixing `Instance` and `Machine` there. Can we choose one word for this?